src/api/middleware/verify_json.ts

```diff
@@ -12,7 +12,7 @@ import StatusCodes from "../utils/status_codes";
  */
 export const verify_username = expressAsyncHandler(
   async (req: Request, res: Response, next: NextFunction) => {
-    const username = req.body.username
+    const username = req.body.username;
     if (!username) {
       res.status(StatusCodes.UNAUTHORIZED).send({
```

Comment:
Just realized use of `send` instead of `json`
replace instances of `.send` with `.json` in this file, we should be replying to the requesting client with JSON

https://expressjs.com/en/4x/api.html#res.json

Reply:
Ah good catch on that one :raised_hands: